### PR TITLE
Add Debian 8 and Ubuntu 16.04 vagrant boxes

### PR DESCRIPTION
**Pull request by AkshitaJha**
Signed-off-by: Akshita Jha <zenith158@gmail.com>

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -478,4 +478,41 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+  #
+  # Debian 8 boxes
+  #
+
+  # Box that installs ODL via its Ansible role on Debian 8
+  # This uses the default repo-based deb install method
+  config.vm.define "debian8_ansible" do |debian8_ansible|
+    # Build Vagrant box based on Debian 8
+    debian8_ansible.vm.box = "debian/jessie64"
+    debian8_ansible.vm.box_version = "=8.5.2"
```
Comment (dfarrell07) [inline Vagrantfile]: Maybe we should build it based on latest, so this doesn't need to be maintained to stay recent?

File: Vagrantfile [hunk 2]
```diff
@@ -478,4 +478,41 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+
+    # Install ODL using the Ansible provisioner
+    debian8_ansible.vm.provision "ansible" do |ansible|
+      # Path to Ansible playbook that installs ODL using ODL's Ansible role
+      ansible.playbook = "provisioning/all_defaults_playbook.yml"
```
Comment (dfarrell07) [inline Vagrantfile]: After you address https://github.com/dfarrell07/ansible-opendaylight/pull/28#pullrequestreview-11056029, you'll need to change this to define/use a playbook that sets deb_repo as install method.

File: Vagrantfile [hunk 3]
```diff
@@ -478,4 +478,41 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+  # This uses the default repo-based deb install method
+  config.vm.define "ubuntu16_ansible" do |ubuntu16_ansible|
+    # Build Vagrant box based on Debian 8
+    ubuntu16_ansible.vm.box = "ubuntu/xenial64"
+    ubuntu16_ansible.vm.box_version = "=20161130.0.0"
+
+    # Install ODL using the Ansible provisioner
+    ubuntu16_ansible.vm.provision "ansible" do |ansible|
+      # Path to Ansible playbook that installs ODL using ODL's Ansible role
+      ansible.playbook = "provisioning/all_defaults_playbook.yml"
```
Comment (dfarrell07) [inline Vagrantfile]: Same deb_repo as install method feedback as above

File: Vagrantfile [hunk 4]
```diff
@@ -478,4 +478,41 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
...
+  #
+  # Ubuntu boxes
+  #
+
+  # Box that installs ODL via its Ansible role on Debian 8
+  # This uses the default repo-based deb install method
+  config.vm.define "ubuntu16_ansible" do |ubuntu16_ansible|
+    # Build Vagrant box based on Debian 8
+    ubuntu16_ansible.vm.box = "ubuntu/xenial64"
+    ubuntu16_ansible.vm.box_version = "=20161130.0.0"
```
Comment (dfarrell07) [inline Vagrantfile]: Same "maybe use latest?" feedback as above

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
Looks great overall, thanks Akshita! Minor feedback inline.

File: Vagrantfile (hunk 1 above)
Comment (AkshitaJha) [reply Vagrantfile]: Done.

File: Vagrantfile (hunk 2 above)
Comment (AkshitaJha) [reply Vagrantfile]: I have added `provisioning/deb_path_install_playbook.yml` and `provisioning/deb_repo_install_playbook.yml`.

**Comment by dfarrell07** [comment]
Perfect Akshita, thanks! Statically reviewed and `vagrant up`'d/poked around, looks good.

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**